spec/models/change_history_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+
+RSpec.describe ChangeHistory do
+  describe "#latest_change_note" do
+    context "change_history is nil" do
+      let(:change_history){ ChangeHistory.new(history: nil) }
+      it "returns nil" do
+        expect(change_history.latest_change_note).to be_nil
+      end
+    end
+
+    context "change_history has one entry" do
+      let(:change_history){
+        ChangeHistory.new(
+          history: [
+            {
+              "public_timestamp" => "2017-10-19T16 =>09 =>23.000+01 =>00",
```

Comment:
There are some strange ` =>` strings in these dates instead of `:`s.

Reply:
Eueew.. don't know what happened there. I've fixed that up.